governor/src/clock/quanta.rs

```diff
@@ -100,8 +100,7 @@ impl Clock for QuantaUpkeepClock {
 
     fn now(&self) -> Self::Instant {
         QuantaInstant(Nanos::from(
-            self.reference
-                .saturating_duration_since(self.clock.recent()),
+            self.clock.recent().saturating_duration_since(self.reference),
```

Comment:
Can you re-run `cargo fmt` and commit the change, please?

Reply:
No problems, reformatted and combined the new advance test as only one Upkeep thread can be running at a time and this seemed the simplest way to avoid issues. 

Thought there might have been a solution to running multiple tests against Upkeep in the Quanta repo, but they only have one too. Didn't catch it the first time around as I use nextest and it had these tests isolated.